### ISSUE TITLE
Ambiguous name issue with `never`

### ISSUE THREAD

**Issue by andys8** [lukewestby/elm-string-interpolate#10]
The package will most likely lead to an error with 0.19.1. Current alphas show that there is an issue with an ambiguous name.

```
-- AMBIGUOUS NAME ----------------------------------- src/String/Interpolate.elm

This usage of `never` is ambiguous:

32|     fromString "\\{\\d+\\}" |> withDefault never
                                               ^^^^^
This name is exposed by 2 of your imports, so I am not sure which one to use:

    Basics.never
    Regex.never

I recommend using qualified names for imported values. I also recommend having
at most one `exposing (..)` per file to make name clashes like this less common
in the long run.

Note: Check out <https://elm-lang.org/0.19.1/imports> for more info on the
import syntax.
```

**Comment by andys8** [comment]
Will be fixed with https://github.com/lukewestby/elm-string-interpolate/pull/9

**Comment by lukewestby** [comment]
@andys8 thank you for reporting. closed by #9 indeed
